src/main/java/com/senkiv/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -38,4 +41,21 @@ public BookDto findById(Long id) {
                 .orElseThrow(() -> new EntityNotFoundException(
                         NO_BOOK_WITH_SUCH_ID.formatted(id)));
     }
+
+    @Override
+    public BookDto updateById(Long id, CreateBookRequestDto bookDto) {
+        Optional<Book> byId = bookRepository.findById(id);
+        if (byId.isPresent()) {
+            Book model = mapper.toModel(bookDto);
+            model.setId(id);
+            bookRepository.save(model);
+            return mapper.toDto(model);
+        }
```

Comment:
Create method `Book update(Book book, CreateBookRequestDto bookDto)` in the mapper